.github/workflows/e2e-test.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       # Conditionally skip WC Blocks tests. Remove/update based on min supported WC version by the blocks checkout plugin.
       - name: Conditionally skip WC Blocks tests
         run: |
-          SKIP_WC_VERSIONS=('5.4.0')
+          SKIP_WC_VERSIONS=('5.6.0')
```

Comment:
This was added because an old version created problems in the E2E tests. Do we still need it for this version?